src/e2e/resources/simple-temple-expected/simple-temple-test-user/dao/dao.go

```diff
@@ -191,3 +246,88 @@ func (dao *DAO) IdentifySimpleTempleTestUser(input IdentifySimpleTempleTestUserI
 
 	return &simpleTempleTestUser, nil
 }
+
+// ListFred returns a list containing every fred in the datastore
+func (dao *DAO) ListFred(input ListFredInput) (*[]Fred, error) {
+	rows, err := executeQueryWithRowResponses(dao.DB, "SELECT id, created_by, field, friend, image FROM fred WHERE parent_id = $1;", input.ParentID)
+	if err != nil {
+		return nil, err
+	}
+
+	fredList := make([]Fred, 0)
+	for rows.Next() {
+		var fred Fred
+		err = rows.Scan(&fred.ID, &fred.Field, &fred.Friend, &fred.Image)
```

Comment:
We select `created_by`, but then don't scan the value into the struct. This works right?

Reply:
Aah, I hadn’t run this without #375 (I rebased it so it could be reviewed sooner), forgetting that the sql updates would copy through to here. Fixed that now. I’ve run it, but in this commit it doesn’t get run yet because the calling of these hooks hasn’t yet happened.